qcodes/instrument_drivers/ZI/ZIHDAWG8.py

```diff
@@ -63,11 +63,12 @@ def __init__(self, name: str, device_id: str, **kwargs) -> None:
         self.warnings_as_errors: List[str] = []
         self._compiler_sleep_time = 0.01
 
-    def snapshot_base(self, update: bool = False,
+    def snapshot_base(self, update: bool = True,
```

Comment:
This silently drops ```params_to_skip_update```, perhaps we should either append `features_code` to whatever is passed or simple make the default value ```('features_code',)```

Reply:
I like appending `features_code` to whatever is passed, rather than having it as a default value.